react-accordion-component/my-app/src/RenderTopics.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export default function RenderTopics(props) {
```

Comment:
It's best to use destructuring to declare your props. It makes it much easier to figure out what props to pass when you use the component in JSX.